Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:12.3.1-alpine as build
+FROM node:14-alpine as build
```

Comment:
Why not also the minor and patch?

Reply:
do we need to be that specific? It means that we've to constantly update the version while we just need this example to use v14; once installed on a project a user may pin a specific node version if they have the requirement.